##Parameter for target_tracking.cpp

##追踪算法比例参数，越大响应越快
kpx_track : 0.6
kpy_track : 0.6
kpz_track : 0.8

##追踪算法最大追踪速度
##取决与使用场景，室内或者第一次实验时，建议选小一点
track_max_vel_x : 0.5
track_max_vel_y : 0.5
track_max_vel_z : 0.5

##追踪算法速度死区
track_thres_vel_x : 0.05
track_thres_vel_y : 0.05
track_thres_vel_z : 0.05

##前后方向是否追踪标志位 1 for track x, 0 for not track x
flag_x : 1

##视觉丢失次数阈值
count_vision_lost : 10

##追踪的前后间隔
delta_x : 1.5

##追踪距离阈值
distance_thres: 0.4



